qt-bugs@ issue : N190509
Trolltech task ID : 190904
applied: yes
author: Rafael Fernández López <ereslibre@kde.org>

Links are assigned a foreground color according to the system current color scheme.

--- a/src/gui/text/qtexthtmlparser.cpp
+++ b/src/gui/text/qtexthtmlparser.cpp
@@ -1031,7 +1031,7 @@
                     && !attributes.at(i + 1).isEmpty()) {
                     hasHref = true;
                     charFormat.setUnderlineStyle(QTextCharFormat::SingleUnderline);
-                    charFormat.setForeground(Qt::blue);
+                    charFormat.setForeground(QApplication::palette().link());
                 }
             }
 
